test_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,68 @@
+************************
+How to test a PR locally
+************************
+
+This guide describes all steps needed to test another author's pull request (hereinafter ``PR``).
+
+It can be useful, particularly, for reviewers and issue authors when they want to check if a PR fixes the reported bug.
+
+.. contents:: Topics
+
+Prepare your environment
+========================
+
+We assume that you use Linux as a work environment (you can use a virtual machine as well) and have ``git`` installed.
+
+1. If possible, make sure that you have installed and started ``docker``. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements, and tests are running properly isolated and in the exact same environments as in CI. You often can also use ``podman`` with the ``docker`` executable shim, so if you have that you probably do not need to install ``docker``. The latest ansible-core development version also supports the ``podman`` CLI program.
```

Comment:
```suggestion
```
Why keep this if nothing in these instructions mentions `docker` or `podman`?

Reply:
Right, PR author should fix the module/tests to pass the CI, tester can see CI results there too, there is no point in running `ansible-test` when you want to **just test** a PR.